mlflow/utils/file_utils.py

```diff
@@ -222,6 +222,14 @@ def render_and_merge_yaml(root, template_name, context_name):
     j2_env = jinja2.Environment(
         loader=jinja2.FileSystemLoader(root, encoding=ENCODING), undefined=jinja2.StrictUndefined
     )
+
+    def from_json(input_var):
+        import json
+
+        with open(input_var, mode="r", encoding="utf-8") as f:
```

Comment:
Out of curiosity, is this set of arguments (reading using `utf-8` encoding etc) copied from somewhere e.g. the ansible `from_json` implementation? It looks reasonable but just wondering

Reply:
Not from Ansible. This is how I do it usually.